core/ui/src/router/index.js

```diff
@@ -44,6 +45,11 @@ const routes = [
     name: "SettingsCluster",
     component: SettingsCluster,
   },
+  {
+    path: "/settings/smarthost",
+    name: "SettingsSmarthost",
+    component: SettingsSmartHost,
```

Comment:
Since Smarthost page is not accessed very often, I suggest to load it dynamically, using `component: () => import(...)` like `SettingsSoftwareRepositories` below

Reply:
done